base/BUILD.gn

```diff
@@ -4016,6 +4024,69 @@ test("base_unittests") {
       "tracing/perfetto_task_runner_unittest.cc",
     ]
 
+    # TODO(cobalt, b/382115610): Re-enable use_partition_alloc
```

Comment:
This looks like more than just partition_alloc stuff, or do all of these files have linking issues with it?

Reply:
a chunk of them do. after adding them one by one for 4-5 files, i just copied over the entire section from 25lts - do iterate faster. We can address these whenever we re-enable use_partition_alloc .. is that okay ?